src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -253,7 +256,7 @@ private void decodeOtherParameter(Position position, int id, ByteBuf buf, int le
                 position.set("workMode", readValue(buf, length, false));
                 break;
             case 81:
-                position.set(Position.KEY_OBD_SPEED, readValue(buf, length, false));
+                position.set(Position.KEY_OBD_SPEED, UnitsConverter.knotsFromKph(readValue(buf, length, false)));
```

Comment:
This is not correct. For OBD speed we actually use kph, as far as I remember.

Reply:
Yes it is in knots! It was strange to me, after testing for OBD speed i have noticed that it was incorect in web, so diging arround i have found comment to KEY_OBD_SPEED that it is in KNOTS don't know why.. also diging around in other protocols there is some which converts kph to knots to store in DB.. if you could replace whole logic to use kph instead of knots it would be great :)

Reply:
Can you please provide specific examples.

Reply:
At first I have noticed that in web page obd speed values shows strange numbers, they are not related to any speed sources available in protocol.. just didn't know why it was wrong.. thought that maybe ID is wrong, or readings from CAN incorrect... then I tried to dig where could be problem. So first i have noticed comment:
https://github.com/traccar/traccar/blob/d834c23f8602fd8f3321d8cb8ff9648b163cdafd/src/main/java/org/traccar/model/Position.java#L90

It shows that value is in knots.. ok thought maybe it's only leftovers from something old.. and why it should be knots ?!? So tried to see which devices uses same value, found couple of them, and only some have conversation in knots from kph 
`FreeMatics, GI2000, XT2400` 

So I Just tried to implement the same thing on my testings.. and OBD speed value start to show correct speed. 

Diggig deeper:

https://github.com/traccar/traccar/blob/d834c23f8602fd8f3321d8cb8ff9648b163cdafd/src/main/java/org/traccar/model/Position.java#L249

again -  knots.
 
and from there i went back to teltonika protocol 
https://github.com/traccar/traccar/blob/d834c23f8602fd8f3321d8cb8ff9648b163cdafd/src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java#L509

so it looks like speed value need to be saved in knots.. sorry didn't look deeper..

Reply:
OK, so the "speed" value should be in knots, but "obd speed" should not.

Reply:
It's just works after using that conversion.. 

Look:
https://github.com/traccar/traccar-web/blob/d67213556dcc87453c0ad5cd6f9e8851c477a2f3/web/app/store/PositionAttributes.js#L227-L232

This is from web frontend, obdspeed, is using data type of speed, also same is for speed value so i am 100% Shure that front end uses knots for speed value.

Also here:
https://github.com/traccar/traccar-web/blob/d67213556dcc87453c0ad5cd6f9e8851c477a2f3/web/app/store/SpeedUnits.js#L18-L35

Knots have factor of 1.. so agen all speed values in database is expected to be in knots!

Reply:
That's an issue in the web app.

Reply:
Omg so what should we do? One value is saved in form where web understands.. another value is not like that.. but both treaded as knots.. some protocol decoder's converts kphToKnots to work with frontend.. anothers leaves as is.. it's strange to me that nobody noticed that?

Reply:
I see two options:
- Change everything to use knots.
- Change everything to use km.

For this specific PR, most places in the code use km/h as the unit for OBD speed, correct? If that's true, then you should also use km/h here.

Reply:
For my opinion it should be saved in km/h, knots used only in boats or planes.  It is possible to change everything to km/h but it will be pain to change saved historical values. It's looks that might bee 50/50 using in knots or kph..

Reply:
In this case it was already km/h.

Reply:
So to work in current situation it need's to be in knots. If it's ok then there will be issue in whole project.

Reply:
Once again, "speed" and "OBD speed" are two completely different concepts in Traccar.

Reply:
Yes speed is stored in column, obdspeed stored as attribute.. so what? web front-end still treats it as speed value, and will use converter to show speed. By default it thinks that it stored in knots.

Reply:
I'm not sure what we are arguing about. I already explained options.

Reply:
Ok, web frontend shows speed values, GPS (main speed value shows correct) value taken from OBD show's incorrect. This is an issue which I would like to fix with along others.. so situation is that web interface treat's any speed to be stored in database as knots. In that sittuation only GPS speed is shown in web site correct becouse it is converted to knots, but obd speed is stored as kmph, so in web frontend it will be converted becouse front-end thinks that obd speed is stored in knots.. i know that it is web problem but to solve that issue we need to store obd value in knots!